API_202404/src/main/java/com/example/api_202404/services/SchoolService.java

```diff
@@ -36,4 +38,37 @@ public Student findStudentBySchoolIdAndStudentId(Long schoolId, Long studentId)
         return null;
 
     }
+
+    public School addSchool(School school) {
+        try {
+            schoolRepository.saveAndFlush(school);
+        } catch (Exception e) {
+            System.out.println("ERROR: " + school.getName() + " school was not added");
+        }
+        return school;
+    }
+
+    public Student addStudentToSchoolbyId(Long schoolId, Student student) {
+        try {
+            School school = findSchoolById(schoolId);
+            school.getStudents().add(student);
+            schoolRepository.saveAndFlush(school);
+        } catch (Exception e) {
+            System.out.println("Failed to add " + student);
+        }
+        return student;
+    }
+
+    public School updateSchoolNameById(Long schoolId, String newName) {
+        School school = null;
+        try {
+            school = findSchoolById(schoolId);
+            String odlName = school.getName();
```

Comment:
odl:)

Reply:
Ir be to, odlName niekur nenaudojamas

Reply:
damm, refactorinimas ne mano striprybė, tiek visko palikau 🤦‍♀️